docs/howto/openstack/keystone/app-creds.md

```diff
@@ -0,0 +1,277 @@
+| expires_at   | None                                                          |
+| id           | c3505060e42542a29ce1b77435112590                              |
+| name         | modest_mccarthy                                               |
+| project_id   | dfc700467396428bacba4376e72cc3e9                              |
+| roles        | creator reader swiftoperator load-balancer_member member      |
+| secret       | qsV9G-qklvbZ8Fz5m7arghW7hHYDRCYWE3vim5dm3-EcXgQn-             |
+|              | pkvrDAOAW5nR7vfj32_HwtQbLjq0TyWM-gETg                         |
+| system       | None                                                          |
+| unrestricted | True                                                          |
+| user_id      | e50719d594f84f16ad13f88da540f762                              |
++--------------+---------------------------------------------------------------+
+```
+
+In the example output above, the `unrestricted` parameter is set to `True`.
+
+> The restricted-by-default policy acts as a safeguard, so compromised application credentials cannot be used for creating other sets of application credentials.
+
+## Using application credentials
+
+One way to use application credentials is by instantiating certain `OS_` variables.
+Then, you may work with the OpenStack CLI as you typically do.
+Let us see how you can go about this, by first creating a new set of application credentials:
+
+```console
+$ PRETTY_GOOD_SECRET="$(pwgen -Bcns 32 1)"
+$ openstack application credential create practical_swanson --secret $PRETTY_GOOD_SECRET
+$ openstack application credential show practical_swanson
+
++--------------+----------------------------------------------------------+
+| Field        | Value                                                    |
++--------------+----------------------------------------------------------+
+| description  | None                                                     |
+| expires_at   | None                                                     |
+| id           | 3e04d504d754445e8b8d503e0e7af3c5                         |
+| name         | practical_swanson                                        |
+| project_id   | dfc700467396428bacba4376e72cc3e9                         |
+| roles        | swiftoperator reader member creator load-balancer_member |
+| system       | None                                                     |
+| unrestricted | False                                                    |
+| user_id      | e50719d594f84f16ad13f88da540f762                         |
++--------------+----------------------------------------------------------+
+```
+
+You need to instantiate four `OS_` variables:
+
+* `OS_AUTH_URL`, which, in our example, is set to `https://fra1.citycloud.com:5000`,
+* `OS_AUTH_TYPE`, which should be set to `v3applicationcredential`,
+* `OS_APPLICATION_CREDENTIAL_ID`, which should be set to the value of `id` (as displayed in the table above),
```

Comment:
Maybe mention that alternatively, one can also use `OS_APPLICATION_CREDENTIAL_NAME`, in which case one doesn't need the generated `id` at all?